redis_signals.go

```diff
@@ -72,8 +72,12 @@ func handleRedisEvent(v interface{}) {
 		OnServerStatusReceivedHandler(notif.Payload)
 	case NoticeGatewayLENotification:
 		OnLESSLStatusReceivedHandler(notif.Payload)
-	default:
+	case NoticeGroupReload:
```

Comment:
I think this is perfectly OK, so I will merge this, but there's a potential case where a reload is triggered by a policy update signal which we might need to keep an eye on in regression testing with the dashboard.